examples/tutorials/08_research_tutorial/1_Research_Introduction.ipynb

```diff
@@ -0,0 +1,2513 @@
+{
```

Comment:
<p><code>Research</code> does not have a plural form</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>